api/handler.go

```diff
@@ -163,6 +166,43 @@ func streamResults(results chan *events.Event, w http.ResponseWriter, r *http.Re
 	}
 }
 
+func requestResults(w http.ResponseWriter, r *http.Request) error {
+	oldUrlString := r.URL.RequestURI()
+	glog.Infof("oldUrlString is %v", oldUrlString)
+	requestElements := clientRegexp.FindStringSubmatch(oldUrlString)
+	if len(requestElements) != 3 {
+		return fmt.Errorf("Could not interpret client request: %v", requestElements)
+	}
+	newUrlString := "http://localhost:8080" + requestElements[1] + "events" + requestElements[2]
```

Comment:
Can you add comments on why we need to map it to localhost and hardcode port number here?